clients/src/test/java/org/apache/kafka/test/TestUtils.java

```diff
@@ -126,6 +128,18 @@ public static File tempDirectory(String prefix) throws IOException {
     }
 
     /**
+     * Create a temporary directory named "test" under /temp
+     * @return  the temporary directory just created.
+     */
+    public static File tempDir() {
```

Comment:
Hmm, not sure about this method:

* It is confusingly called `tempDir` even though we have a `tempDirectory` in the same class.
* It uses a different prefix than `tempDirectory` (`test` instead of `kafka-`) but the latter seems to be better.
* The documentation also seems to be wrong in stating that the directory name will be "test". Only the prefix will be that, the directory name will have some random elements in it.
* It passes the temp directory as the parent, but it seems better to let `Files.temporaryDirectory` choose the OS temp directory.
* It converts IOException to RuntimeException. This one seems like a good thing. Maybe we should change `temporaryDirectory` to do it instead?